app/code/Magento/Store/Api/StoreCreateManagementInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Magento\Store\Api;
+
+interface StoreCreateManagementInterface
```

Comment:
I don't think the model need an interface currently. Let's keep it internal implementation for now and just work with the class